test/e2e/main.grpc.test.ts

```diff
@@ -267,6 +270,94 @@ describe('grpc/main', () => {
     });
   });
 
+  describe('configuration', () => {
+    beforeEach(async () => {
+      // Reset the Configuration API
+      await DockerUtils.executeDockerCommand("dapr_redis redis-cli MSET myconfigkey1 initialvalue||1");
+      await DockerUtils.executeDockerCommand("dapr_redis redis-cli MSET myconfigkey2 initialvalue||1");
+      await DockerUtils.executeDockerCommand("dapr_redis redis-cli MSET myconfigkey3 initialvalue||1");
+    })
+
+    it('should be able to get the configuration items', async () => {
+      // await server.configuration.subscribe();
+      const conf = await client.configuration.get("config-redis", ["myconfigkey1", "myconfigkey2", "myconfigkey3"]);
+      expect(conf.items.length).toEqual(3);
+
+      expect(conf.items.map(i => i.key).indexOf("myconfigkey1")).toBeGreaterThan(-1);
+      expect(conf.items.map(i => i.key).indexOf("myconfigkey2")).toBeGreaterThan(-1);
+      expect(conf.items.map(i => i.key).indexOf("myconfigkey3")).toBeGreaterThan(-1);
+
+      expect(conf.items.filter(i => i.key == "myconfigkey1")[0].value).toEqual("initialvalue");
```

Comment:
Shall we use different initial values? Like `initialvalue1`, `initialvalue2`, etc.?

Reply:
Yep, will make it clearer